cypress/e2e/homepage-spec.cy.js

```diff
@@ -24,20 +22,23 @@ describe('homepage spec', () => {
     cy.get('.choose-category-title').contains('h1', 'Choose a category from the options below');
   });
   describe('categories on homepage leading to their respective pages', () => {
-    const categories = ['Exercise', 'Cleaning', 'Organization', 'Work', 'Mental Care', 'Health'];
+    const categories = {
+      exercise: 'Exercise',
+      cleaning: 'Cleaning',
+      organization: 'Organization',
+      work: 'Work',
+      mentalCare: 'Mental Care',
+      health: 'Health',
+    };
     it('should render 6 categories', () => {
-      const cypressCategories = cy.get('.categories .category');
-      categories.forEach((cat) => {
-        cypressCategories.should('contain', cat);
+      Object.entries(categories).forEach(([catID, catValue]) => {
+        cy.get(`#${catID}`).should('contain', catValue);
```

Comment:
Could you explain these 2 lines please?

Reply:
I changed the categories variable to be an object instead of an array. It allows me to test a bit more specifically. I also added the IDs to the category elements on the home page for easier selecting when testing.